functions/zram.bats

```diff
@@ -82,22 +82,20 @@ check_zram_removal() {
   if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
   [ "$status" -eq 0 ]
   echo -e "# ${COL_GREEN}$(timestamp) [openHABian] Availability of zram mounts verified.${COL_DEF}" >&3
-  run init_zram_mounts "uninstall" 3>&-
```

Comment:
this is from your zram patch (which I've only noticed by coincidence as you mentioned it in NEWS.md)
and like above this should not be in this PR

Reply:
That is mostly true but in this case it is related to CI, I know this is a jumbled patch but it was essentially a bunch of cumulated patches that I discovered while doing CI upgrades.